SurveyApp/Views/AnswerView/MultipleChoiceAnswerView.swift

```diff
@@ -32,7 +32,7 @@ class MultipleChoiceAnswerView: UIView {
             $0.leading.equalToSuperview().inset(10.0)
             $0.trailing.equalToSuperview().inset(20.0)
             $0.top.equalToSuperview().inset(50.0)
-            $0.height.equalToSuperview().inset(50.0)
+            $0.height.lessThanOrEqualTo(300.0)
```

Comment:
Can you check if you need to add `$0.height.lessThanOrEqualToSuperview().inset(50.0)` for smaller device.

Reply:
@blyscuit  I tested `$0.height.lessThanOrEqualToSuperview().inset(50.0)` with iPhone SE but the table view doesn't display.

This is result when use `$0.height.lessThanOrEqualTo(300.0)` on iPhone SE and add `$0.bottom.equalToSuperview().inset(20.0)` to have some space between next button and table:

I'll update the code if you agree with this approach, please let me know 🙇‍♀️

<img width="342" alt="Screen Shot 2021-07-08 at 6 10 30 PM" src="https://user-images.githubusercontent.com/45258998/124912131-c93cb280-e017-11eb-92ca-95f9e5e78e14.png">

Reply:
you are right, it should've been `$0.bottom.lessThanOrEqualToSuperview().inset(20.0)`

Reply:
@blyscuit Sorry, I misunderstood.
It's looks better with `$0.height.lessThanOrEqualToSuperview().inset(50.0) `:
I will update the code, please help check again. Thank you 🙏

<img width="357" alt="Screen Shot 2021-07-08 at 6 21 02 PM" src="https://user-images.githubusercontent.com/45258998/124913384-41f03e80-e019-11eb-96c7-994e886d8e5d.png">

Reply:
> you are right, it should've been `$0.bottom.lessThanOrEqualToSuperview().inset(20.0)`

@blyscuit  This approach got the same result and work fine on both small devices and large devices. I updated the code to this approach instead. 
Please help check. Thank you so much🙇‍♀️

Reply:
I mean, it should've been the same right, I expect small device will show smaller table 🤔 This is from iPod 7.
![Simulator Screen Shot - iPod touch (7th generation) - 2021-07-08 at 18 35 30](https://user-images.githubusercontent.com/6356137/124915049-47e71f00-e01b-11eb-9b3f-f6580b4c6a08.png)